extensions/2.0/Khronos/KHR_audio/schema/source.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Source",
+  "type": "object",
+  "description": "A source audio clip with defined properties about the behavior of that audio.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+    "autoPlay": {
+        "description": "Whether or not to play the specified audio when the glTF is loaded.",
+        "type": "boolean",
+        "default": false
+    },
+    "gain": {
+        "description": "Unitless multiplier against original audio file volume for determining audio source loudness.",
+        "type": "number",
+        "minimum": 0.0,
+        "default": 1.0
+    },
+    "loop": {
+        "description": "Whether or not to loop the specified audio when finished.",
+        "type": "boolean",
+        "default": false
+    },
```

Comment:
You made a mistake with this schema.

* In the README, `"loop"` and `"gain"` are a part of the audio emitter, not the audio source.
* The `"autoPlay"` property is not defined in the README.

Reply:
The audio source is intended to control audio playback, so `gain`, `loop`, and `autoPlay` would be on the `audioSource`. The json schema is correct and the README needs to be changed.

`playing` was changed back to `autoPlay` in the latest draft of the spec. So the schema is also correct there as well.

Looks like we have a few things to fix in the README and example.

Reply:
todo: @antpb match readme to the schema